components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add.jsp

```diff
@@ -187,6 +187,19 @@
 		                                   type="text" /></td>
 		                    </tr>
 		                    <% } %>
+                            <tr>
```

Comment:
The default values set for token expiry need to be read from the identity.xml. We cannot simply add the default values we ship since its possible change the default values.

For example, I go and change the default user access token validity period to 300s in identity.xml. Upon restart I expect all the apps that I create newly to have 300s as access token validity not 3600s

Reply:
Also, it would be better to let the users define these expiry time in seconds since that is how we define in identity.xml

Reply:
UI validation is done